Resources/public/css/layout.css

```diff
@@ -355,6 +355,10 @@ body.fixed .content-header .navbar.stuck {
     border-radius: 0;
 }
 
+.box .nav-stacked > li {
```

Comment:
As I said, I don't think it's a bug, this is the default behavior in bootstrap.

Overflow management may be decided by us, not by bootstrap.

I don't know if apply it on generic navs would not make some side effects.

@rande what do you think?